__tests__/config/integration/jest.config.json

```diff
@@ -24,5 +24,5 @@
         "unit",
         "__system__"
     ],
-    "testRegex": "(test|spec)\\.(ts)"
+    "testRegex": "__integration__.*(test|spec)\\.(ts)"
```

Comment:
Is this one-line change all that was needed to stop running our unit tests twice? 😋

Reply:
it was enough to get most integration tests running.
For instance there are some very old integration tests that are not contained in a `__integration__` folder
Ex: `__tests__/src/example_clis/with_experimental/ExperimentalExample.spec.ts`

Should we move them around so that they get picked up as well ?

Reply:
Looks like this was done in https://github.com/zowe/imperative/pull/697? 😋

Reply:
yes.
I'm hesitant to remove it as it may cause a merge conflict with #697
😢